ReportUtilityBills/tests/utility_bills_test.rb

```diff
@@ -1199,8 +1199,11 @@ def _get_actual_bills(bills_csv)
       next if line.strip.empty?
 
       key, value = line.split(',').map { |x| x.strip }
+      next if (1..12).to_a.any? { |month| key.include?(": #{month}:") }
+
       actual_bills[key] = Float(value)
```

Comment:
Rather than ignoring monthly outputs in the test, please check that the sum of the monthly results equals the expected annual result too.